lib/teslamate/import/line_parser.ex

```diff
@@ -76,6 +76,9 @@ defmodule TeslaMate.Import.LineParser do
       {"id", _val} ->
         Map.put(acc, "id", :rand.uniform(65536))
 
+      {"vehicle_id", ""} ->
+        Map.put(acc, "vehicle_id", "1")
```

Comment:
hard coding vehicle id will not be merged

Reply:
Is there any acceptable alternative? This is used in lieu of a default value.

As mentioned in the description, this what is recommended/suggested in the linked comment.

Reply:
Proper format input file with correct vehicle id, as suggested in the linked comment. Hardcoding vehicle id to import to 1 will result in overwriting old data if that vehicle id exists before.

Reply:
> Proper format input file with correct vehicle id, as suggested in the linked comment. 

Isn’t a "proper format input file" for this TeslaFi Importer what is directly exported from TeslaFi without manual modifications?

> Hardcoding vehicle id to import to 1 will result in overwriting old data if that vehicle id exists before.

Doesn’t the current code ignores any input data if there is already some dated before or on the day of the current entry/event?
At worse, using 1 as the default value would add data to the wrong vehicle, but should not overwrite anything.

Reply:
@JakobLichterfeld should this use `:rand.uniform(65536)` like the `id` field?

Reply:
No, as the vehicle id is given by tesla

Reply:
I don’t know about tesla’s API, but the TeslaFi exports do not report it.

Reply:
@ithinuel Having imported many TeslaFi's, I confirm that it is tedious to fill in the vehicle_id column in each file.

This vehicle_id corresponds to the car_id of the Teslamate "cars" table.

@JakobLichterfeld Is it OK to use a new environment variable to specify the car ID for which TeslaFi data will be imported?

I should ends with something like that: 
```
      {"vehicle_id", ""} ->
        Map.put(acc, "vehicle_id", System.get_env("TESLAFI_IMPORT_VEHICLE_ID", "1"))
```

Reply:
@jlestel  I (finally) applied your suggestion, and rebased on master.

It looks like CI is happy so the PR is ready for review again :)